test/channels/retro_channel_test.exs

```diff
@@ -46,13 +46,6 @@ defmodule RemoteRetro.RetroChannelTest do
 
   describe "pushing `proceed_to_next_stage` with a stage of 'action-item-distribution'" do
     setup [:join_the_retro_channel]
-
```

Comment:
why are we deleting this test? let's augment the expectation such that it expects it to be sent to the dev user's email.